gr-qtgui/lib/freq_sink_c_impl.cc

```diff
@@ -410,6 +419,22 @@ void freq_sink_c_impl::buildwindow()
     if (d_wintype != filter::firdes::WIN_NONE) {
         d_window = filter::firdes::window(d_wintype, d_fftsize, 6.76);
     }
+
+    if (!d_window_normalize) {
+        return;
+    }
+    // Calculate power of the window as 1/N * sum of squares
+    const float power =
+        boost::accumulate(
+            d_window | boost::adaptors::transformed([](const float x) { return x * x; }),
+            0.0) /
```

Comment:
Can be replaced by `std::inner_product()` (or `std::accumulate()` + lambda).

Reply:
Thanks, I used that.